elixir/apps/domain/lib/domain/policies.ex

```diff
@@ -80,6 +80,15 @@ defmodule Domain.Policies do
     end
   end
 
+  def new_policy(attrs, %Auth.Subject{} = subject) do
+    required_permissions =
+      {:one_of, [Authorizer.manage_policies_permission()]}
+
+    with :ok <- Auth.ensure_has_permissions(subject, required_permissions) do
```

Comment:
There is no need to check permissions here, this will also violate the function return (it will be either a changeset or error tuple, which is not idiomatic),

Reply:
This has been updated